### PR TITLE
[CS] Code Style round 1 fixes for libraries/scr

### PR DESCRIPTION
**Pull request by photodude**
Pull Request for Issue  code style fixes

### Summary of Changes
- No blank line found after control structure
- Please consider an empty line before the if statement
- Please consider an empty line before the foreach statement
- Expected 1 space before "?"; 0 found
- Expected 1 space after ":"; 0 found
- Please consider an empty line before the return statement
- Fix some BlockComments

[Automatically fixed with Joomla code standards 2.0.0 PHPCS2-alpha2 fixers](https://github.com/joomla/coding-standards/releases/tag/2.0.0-alpha2)

None of the manual only fixes have been applied

### Testing Instructions
Merge by code review

### Expected result
code style has been applied as listed above, old code style testing on drone does not error.

### Actual result
code style had not been applied. [Autofixers from the Joomla code standards 2.0.0 PHPCS2 alpha2](https://github.com/joomla/coding-standards/releases/tag/2.0.0-alpha2) were used to implement fixable code style

### Documentation Changes Required
none

### CODE REVIEW

File: libraries/src/Environment/Browser.php [hunk 1]
```diff
@@ -264,8 +266,11 @@ public function match($userAgent = null, $accept = null)
 
 				list ($this->majorVersion, $this->minorVersion) = explode('.', $version[1]);
 
-				/* Due to changes in Opera UA, we need to check Version/xx.yy,
-				 * but only if version is > 9.80. See: http://dev.opera.com/articles/view/opera-ua-string-changes/ */
+				/**
+				 * Due to changes in Opera UA, we need to check Version/xx.yy,
+				 * but only if version is > 9.80. See: http://dev.opera.com/articles/view/opera-ua-string-changes/
+				 */
+
```
Comment (Quy) [inline libraries/src/Environment/Browser.php]: Remove blank line?

File: libraries/src/Environment/Browser.php [hunk 2]
```diff
@@ -264,8 +266,11 @@ public function match($userAgent = null, $accept = null)
 
 				list ($this->majorVersion, $this->minorVersion) = explode('.', $version[1]);
```
Comment (Quy) [inline libraries/src/Environment/Browser.php]: No space after `list`?

Comment (photodude) [reply libraries/src/Environment/Browser.php]: Correct, something should catch and fix that. I'm not sure why it didn't. 
Method signatures don't have a space before the opening parenthesis.
Fixed

File: libraries/src/Environment/Browser.php (hunk 1 above)
Comment (photodude) [reply libraries/src/Environment/Browser.php]: fixed

**Comment by Quy** [comment]
I have tested this item :white_check_mark: successfully on d9e944bd5ec512599c405a608ffae222b52be7c4<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/19177">issues.joomla.org/tracker/joomla-cms/19177</a>.</sub>

File: libraries/src/Environment/Browser.php [hunk 3]
```diff
@@ -242,15 +242,17 @@ public function match($userAgent = null, $accept = null)
 				$this->mobile = true;
 			}
 
-			// We have to check for Edge as the first browser, because Edge has something like:
-			// Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/51.0.2704.79 Safari/537.36 Edge/14.14393
+			/**
```
Comment (Quy) [inline libraries/src/Environment/Browser.php]: With multi lines comment, it should be `/*` and not `/**`.

Comment (photodude) [reply libraries/src/Environment/Browser.php]: manual change, when I changed the comment style, I was thinking it was the same as the docblock style

File: libraries/src/Application/CliApplication.php [hunk 4]
```diff
@@ -60,6 +60,7 @@ public function __construct(Cli $input = null, Registry $config = null, \JEventD
 		{
 			$this->close();
 		}
+
```
Comment (Quy) [inline libraries/src/Application/CliApplication.php]: Delete line 64 if not done in a different PR and other instances in this PR.

Comment (photodude) [reply libraries/src/Application/CliApplication.php]: I hate when I miss the minor find and replace errors link this... should be fixed now

File: libraries/src/Application/WebApplication.php [hunk 5]
```diff
@@ -451,24 +451,20 @@ protected function compress()
 			if (($supported[$encoding] == 'gz') || ($supported[$encoding] == 'deflate'))
 			{
 				// Verify that the server supports gzip compression before we attempt to gzip encode the data.
-				// @codeCoverageIgnoreStart
-				if (!extension_loaded('zlib') || ini_get('zlib.output_compression'))
+						if (!extension_loaded('zlib') || ini_get('zlib.output_compression'))
```
Comment (Quy) [inline libraries/src/Application/WebApplication.php]: Fix indentation.

File: libraries/src/Application/WebApplication.php [hunk 6]
```diff
@@ -451,24 +451,20 @@ protected function compress()
...
-				// @codeCoverageIgnoreEnd
 
 				// Attempt to gzip encode the data with an optimal level 4.
 				$data = $this->getBody();
 				$gzdata = gzencode($data, 4, ($supported[$encoding] == 'gz') ? FORCE_GZIP : FORCE_DEFLATE);
 
 				// If there was a problem encoding the data just try the next encoding scheme.
-				// @codeCoverageIgnoreStart
-				if ($gzdata === false)
+						if ($gzdata === false)
```
Comment (Quy) [inline libraries/src/Application/WebApplication.php]: Fix indentation.

File: libraries/src/Application/WebApplication.php [hunk 7]
```diff
@@ -736,6 +735,7 @@ public function setHeader($name, $value, $replace = false)
 		 * If ($keys && !in_array...) it's a multiple value header
 		 */
 		$single = in_array($name, $this->singleValueResponseHeaders);
+
```
Comment (Quy) [inline libraries/src/Application/WebApplication.php:738]: Fix multi line on line 732

Comment (photodude) [reply libraries/src/Application/WebApplication.php:738]: fixed

**Comment by zero-24** [comment]
Merged on review. Thanks 👍

**Comment by photodude** [comment]
Thanks for the merge @zero-24
